src/cap-description.adoc

```diff
@@ -85,44 +85,81 @@ superfluous when the capability does not grant either <<r_perm>> or <<w_perm>>.
 Therefore, it is only possible to encode a subset of all combinations.
 
 .Permissions widths depending on XLENMAX
-[#perms_bit_width,options=header,align="center",width="55%"]
+[#perms_bit_width,options=header,align="center",width="75%",cols="^1,^3,5"]
 |==============================================================================
-^| XLENMAX ^| Permissions width
-^| 32      ^| {cap_rv32_perms_width}
-^| 64      ^| {cap_rv64_perms_width}
+^| XLENMAX ^| Permissions width      ^| Comment
+^| 32      ^| {cap_rv32_perms_width} ^| Encodes some combinations of 5 permission bits and the Mode bits
+^| 64      ^| {cap_rv64_perms_width} ^| Separate bits for each permission, and the Mode bit is not included
```

Comment:
I would like to align RV64 and RV32 so that, at least for the base ISA, they have the same format.

This would mean that as people start writing software for CHERI it can be transportable between the two. Even for RV64, saving bits is also a great idea. There are plenty of good ideas floating about on how to use extra bits: MTE-like colours, N-bit local/global etc. It seems a shame to waste them just because we were never forced to think about it until RV32.

It will hopefully also simplify the spec somewhat to have fewer differences.

Reply:
Conceptually I agree that having them the same would be ideal, but I'd argue we want RV32 to be like RV64 and allow all combinations. Since that is not feasible due to the limited number of bits available I believe we should just accept that they will be different.